events/wc_culture_conversion_events.txt

```diff
@@ -0,0 +1,185 @@
+
+namespace = WCMLT
+
+# ###########################################
+# # Province conversions                    #
+# ###########################################
+
+# Human province in Barrens/Dustwallow converts to Theramorean
+province_event = {
+	id = WCMLT.1
+	desc = EVTDESC_WCMLT_1
+	picture = GFX_evt_lordaeron_coast
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = human_group
+		NOT = { culture = theramore }
+		OR = {
+			region = world_kalimdor_barrens		 
+			region = world_kalimdor_dustwallow
+			region = island_theramore
+			region = island_alcaz
+		}
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_1
+		culture = theramore
+	}
+}
+
+# Orcish province in Kalimdor converts to Durotaric
+province_event = {
+	id = WCMLT.2
+	desc = EVTDESC_WCMLT_2
+	picture = GFX_evt_durotar
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = orc_group
+		NOT = { culture = durotaric }
+		region = world_kalimdor_durotar
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_2
+		culture = durotaric
+	}
+}
+
+# Human province in Tanaris/Uldum converts to Wastewander
+province_event = {
+	id = WCMLT.3
+	desc = EVTDESC_WCMLT_3
+	picture = GFX_evt_sand
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = human_group
+		NOT = { culture = wastewander }
+		OR = {
+			region = world_kalimdor_tanaris		 
+			region = world_kalimdor_uldum
+		}
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_3
+		culture = wastewander
+	}
+}
+
+# Human province in Kalimdor converts to Western
+province_event = {
+	id = WCMLT.4
+	desc = EVTDESC_WCMLT_4
+	picture = GFX_evt_forest
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = human_group
+		region = world_kalimdor
+		NOT = { culture = western
```

Comment:
Since there's more than one "colonial" culture it makes sense to add them all here. Otherwise, it'll be impossible to convert these lands as ruler of other "colonial" culture. I would add colonial_culture_trigger scripted trigger.
Here's article about what it's and how to do it...
https://ck2.paradoxwikis.com/Scripting#Scripted_block

Reply:
`colonial_culture_trigger = {
	OR = {
		culture = theramorean
		culture = wastewander
		culture = western
		culture = durotaric
	}
}`

Will this do?